backend/apps/content/views.py

```
@@ -290,6 +290,18 @@ def get(self, request, pk):
 
         return response_obj
 
+class LessonAccessCheckView(APIView):
+    """
+    Check if user can access a lesson.
+    """
+    permission_classes = [IsLessonUnlocked]
+    
+    def get(self, request, slug):
+        return Response({
```

Ruff (0.15.20)

[error] Undefined name `Response`

(F821)